lib/get-client-config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = config => {
+  // Form https://github.com/npm/npm/blob/master/lib/cache/caching-client.js#L194
```

Comment:
maybe add permanent link? https://github.com/npm/npm/blob/d081cc6c8d73f2aa698aab36605377c95e916224/lib/cache/caching-client.js#L194

When you press `y` on any file page, it will redirect to permalink

Reply:
Nice, I didn't know that :)

Reply:
Done